internal/servers/controller/handlers/targets/target_service.go

```diff
@@ -45,6 +46,10 @@ import (
 	"google.golang.org/protobuf/types/known/wrapperspb"
 )
 
+const (
+	credentialDomain = "credential"
```

Comment:
We're using both `credential`, `host` and `target` domains in the target service, is this intentional?